ui/src/authentication/pages/Login/Login.js

```diff
@@ -35,28 +47,181 @@ class LoginPage extends Component {
+                      columns={2}
+                      className={'default-margin'}
+                    >
+                      <Image
+                        src={
+                          'https://inveniosoftware.org/static/img/logo-invenio-ils.svg?h=da7ca04b'
+                        }
+                        size="small"
+                        centered
+                      />
+                      <Grid.Row>
+                        <Grid.Column stretched width={8} textAlign="right">
+                          <Header size="massive" className="inline-block">
+                            Hello!
+                          </Header>
+                        </Grid.Column>
+
+                        <Grid.Column width={8} textAlign="left">
+                          <Grid.Row>
+                            <Grid.Column width={16}>
+                              <Header
+                                as="h5"
+                                className="inline-block greetings"
+                              >
+                                Ciao!
+                              </Header>{' '}
+                              <Header
+                                as="h5"
+                                className="inline-block greetings"
+                              >
+                                Γεια σας!
+                              </Header>{' '}
+                              <Header
+                                as="h5"
+                                className="inline-block greetings"
+                              >
+                                Hej!
+                              </Header>
+                            </Grid.Column>
+                            <Grid.Column>
+                              <Header
+                                as="h5"
+                                className="inline-block greetings"
+                              >
+                                Cześć!
+                              </Header>{' '}
+                              <Header
+                                as="h5"
+                                className="inline-block greetings"
+                              >
+                                Salut!
+                              </Header>{' '}
+                              <Header
+                                as="h5"
+                                className="inline-block greetings"
+                              >
+                                Alo!
+                              </Header>
+                            </Grid.Column>
+                          </Grid.Row>
+                        </Grid.Column>
+                      </Grid.Row>
+                    </Grid>
+
+                    <p>
+                      To sign in, choose your preferred method in the right
+                      panel.
+                      <br />
+                      <Link className="alternative" to={FrontSiteRoutes.home}>
+                        main page
```

Comment:
I think there is an issue here no? Maybe `Go to the homepage`?

Reply:
I plan to place it in the bottom left corner, because it is the only navigation link back to the website, and it is a good practice not to have "holes" in the navigation